### PR TITLE
#163383194 Implement Users should be able to share articles

### PR DESCRIPTION
**Pull request by missvicki**
#### What does this PR do?

 Implement Users should be able to share articles across different channels

#### Description of Task to be completed?

- write tests to get sharable links for Facebook, Twitter, Google and email
- write method creates links that will be used to post an article to
    Facebook and Google Plus, or share an article as a tweet on Twitter
    and also share the article through email to another email user

#### How should this be manually tested?

* Fetch and checkout this branch `ft-bookmark-article-163383195`
* Create a virtual environment using `python3 -m virtualenv venv`  and activate it using `source venv\bin\activate`
* Install requirements using `pip install -r requirements.txt`
* Start the app `python manage.py runserver`
* On postman, Retrieve articles using the url `localhost:8000/api/v1/articles/`
* Under share_links for a single article, copy and paste either the twitter link, or facebook link, or google plus link in your browser to test them.
* To run tests, type this in the terminal: `python manage.py test`

#### What are the relevant pivotal tracker stories?

[#163383194](https://www.pivotaltracker.com/story/show/163383194)

#### Screenshots 
screenshot 2019-02-20 at 12 38 44

### CODE REVIEW

File: authors/apps/articles/utils/utils.py [hunk 1]
```diff
@@ -46,3 +47,29 @@ def get_article_read_time(body):
...
+    url = request.build_absolute_uri(reverse('articles:article-details',
+                                             kwargs={'slug': obj.slug}))
+    share_links['google_plus'] = 'https://plus.google.com/share?url='+url
+    share_links['twitter'] = 'https://twitter.com/home?status='+url
+    share_links['facebook'] = 'https://www.facebook.com/sharer/\
+sharer.php?u='+url
+
+    subject = obj.title
+    body = f"""I am sharing this article, '{subject}' \
+with you. Click this link:{url}"""
```
Comment (codjoero) [inline authors/apps/articles/utils/utils.py]: Good work @missvicki , Am wondering why you had to use the doc-string quotes in the f-string

**Comment by fahdjamy** [comment]
Thanks for the simple and good implementation of the task.
Do you mind just dealing with the pep8 white space

**Comment by missvicki** [comment]
> Thanks for the simple and good implementation of the task.
> Do you mind just dealing with the pep8 white space

working on this.

**Review by codjoero — CHANGES_REQUESTED** [review]

File: authors/apps/articles/utils/utils.py (hunk 1 above)
Comment (missvicki) [reply authors/apps/articles/utils/utils.py]: I wanted to be able to use the qoutation e.g. 'this is my subject' within the body message

**Review by codjoero — APPROVED** [review]
LGTM

**Review by karuhanga — APPROVED** [review]
LGTM

**Review by oma0256 — APPROVED** [review]
Good work @missvicki

**Review by pbkabali — APPROVED** [review]

**Review by malep2007 — COMMENTED** [review]
You could do the same thing here, with this PR to improve the description.

**Comment by malep2007** [comment]
@missvicki some merge conflicts here.

**Comment by missvicki** [comment]
> @missvicki some merge conflicts here.

Resolved

**Review by fahdjamy — APPROVED** [review]

**Pull request merged**